### ISSUE TITLE
Invalid Gurobi license

### ISSUE THREAD

**Issue by nslosse** [JuliaOpt/Gurobi.jl#299]
Hello,

I have installed a fresh version of Gurobi.jl and JuMP for Julia Version 1.3.1 on a linux server.

When making call to Gurobi 9.0 from Julia, I bump on the following error message :

ERROR: LoadError: Invalid Gurobi license

After some research on the internet, I have checked that :  

- my Gurobi license file was valid (N.B. this is an evaluation licence)
- all environment variable are properly set
- gurobi works well on its own or when used embedded in python.

Do you have any idea what may cause this error ?

Thank you in advance,

Nicolas Slosse

**Comment by odow** [comment]
Does your license support Gurobi 9.0? Did you set the `GRB_LICENSE_FILE` environment variable? You can check with `ENV["GRB_LICENSE_FILE"]`.

From Gurobi.jl point of view, Gurobi.jl is installed and able to communicate with the binary.

**Comment by nslosse** [comment]
Hi Odow, I confirm that both the environment variable is set that my license file is OK for gurobi 9.0 (which I can run in standalone mode).

This is very strange.

Regards,

Nicolas

**Comment by odow** [comment]
Oh. I missed the server part. You probably need to use https://github.com/JuliaOpt/Gurobi.jl#reusing-the-same-gurobi-environment-for-multiple-solves.

```Julia
const env = Gurobi.Env()  # Do ONE license check
model = Model(() -> Gurobi.Optimizer(env))
```

**Comment by nslosse** [comment]
Hi Odow,

Thanks, that was indeed the issue (I just got the same answer from the Gurobi's helpdesk).

Many thanks,

Nicolas

**Comment by odow** [comment]
Glad you got it sorted.

**Comment by ccoffrin** [comment]
For anyone that may come across this tread, you may run into issues if running the same script multiple times, for example while doing model development.  In that case, I have used the following approach for defining the `env` constant,
```
if !(@isdefined env)
    const env = Gurobi.Env()
end
```

**Comment by mberat** [comment]
Hi, 
I'm having the same issue. Although, I did not have this issue when I worked with JuMP using Gurobi solver few months ago.
As a matter of fact, I don't have this issue when I use Julia/JuMP from R currently. (Yes there is a package in R JuliaCall that allows me to run Julia using R.)
When I run `ENV["GRB_LICENSE_FILE"]` I get the following error:
`Stacktrace:`
`[1] (::getfield(Base, Symbol("##419#420")))(::String) at .\env.jl:76`
`[2] access_env(::getfield(Base, Symbol("##419#420")), ::String) at .\env.jl:14`
`[3] getindex(::Base.EnvDict, ::String) at .\env.jl:76`
`[4] top-level scope at none:0`

Any help would be great!
Thanks
